### ISSUE TITLE
Assembly is not strongly named (signed)

### ISSUE THREAD

**Issue by dahall** [Vereyon/HtmlRuleSanitizer#16]
In order to use this assembly in Visual Studio Extensions, it must be signed.

**Comment by cakkermans** [comment]
Hi David,

Do you really need strong naming? https://stackoverflow.com/questions/16447324/do-visual-studio-extensions-vsix-need-to-be-strong-named

I have had some mixed experiences with using strong named assemblies in situations where there were version conflicts. JSON.NET was an example and I was not alone.

The tide seems to be to avoid strong naming.

Any thoughts?

**Comment by dahall** [comment]
Yes, because I'm depending on other assemblies that are signed and you can't mix. I have deployed multiple libraries on NuGet and have strongly named them all without any issue for many years with over a million downloads on one of them.
